install.md

```diff
@@ -122,6 +122,14 @@ Skopeo has not yet been packaged for Windows. There is an [open feature
 request](https://github.com/containers/skopeo/issues/715) and contributions are
 always welcome.
 
+## GO ENV settings
+
+In order for the system libraries to get compiled during the build process make sure that ```CGO_ENABLED='1'```.
+
+```bash
+go env -w CGO_ENABLED='1'
```

Comment:
https://pkg.go.dev/cmd/cgo says that CGo is the default _if a C compiler is available_. Maybe just show how to determine that it is enabled, without providing detailed instructions on possible ways to do that? 

Also, perhaps, “alternatively, use the `containers_image_openpgp` build tag, if the downsides are acceptable”.